### PR TITLE
Change NOT WITHOUT_DEPENDENCIES to DOWNLOAD_DEPENDENCIES

### PR DESCRIPTION
**Pull request by bencoman**
Avoid negative logic when possible.
(I'm continually finding "NOT WITHOUT_DEPENDENCIES" hard to parse)
Tested on configuration "x86-Debug-StackVM-clang"

### CODE REVIEW

**Comment by guillep** [comment]
I agree with the name change, I was planning to make a pass on the naming once PR https://github.com/pharo-project/opensmalltalk-vm/pull/127 passes. But if you want to push it, I'm ok with it!

**Comment by bencoman** [comment]
That was an accidental close.  I think this is fairly simple and would be good to get in front of the merge into "headless".

**Comment by bencoman** [comment]
btw, it also clears a few Cmake build errors with duplicate variables in CMakeSettings.json. 
e.g...  ```cmakeCommandArgs": " -DFLAVOUR=StackVM -DGENERATED_SOURCE_DIR=stack32",```

**Comment by bencoman** [comment]
> btw, it also clears a few Cmake build errors with duplicate variables in CMakeSettings.json.
> e.g... cmakeCommandArgs": " -DFLAVOUR=StackVM -DGENERATED_SOURCE_DIR=stack32",

That said... I'm not sure if I've cheated by turning off any feature in CMakeSettings.json that you want on by default.
@guillep Can you check each build with the defaults you want before integrating into main `headless` branch.

sidebar: what is the best way to manage personal CMakeSettings.json settings - so my config don't continually pollute and fight with yours?

**Comment by guillep** [comment]
> That said... I'm not sure if I've cheated by turning off any feature in CMakeSettings.json that you want on by default.
> sidebar: what is the best way to manage personal CMakeSettings.json settings - so my config don't continually pollute and fight with yours?

Well, actually my idea behind committing the `CMakeSettings.json` file was to have some kind of template for Visual Studio users to have... But if it creates more noise than what it solves, maybe we can remove it. I see you opened https://github.com/pharo-project/opensmalltalk-vm/issues/138, we can continue this part of the conversation there?

> @guillep Can you check each build with the defaults you want before integrating into main `headless` branch.

Well, that's why this branch is taking so long to integrate, but I'll get there today or tomorrow! :).
I spend several hours last week logging in and out of OSX, Windows (on cygwin!) and Linux to manually test and check everything works as expected there too (if you check I had to do some little adaptations to the vmmaker download changes you did mainly to make it work out of source and on cygwin).
